src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -765,30 +801,72 @@ void CommandRequestImpl::AddDissalowedParameterToInfoString(
 
 void CommandRequestImpl::AddDisallowedParametersToInfo(
     smart_objects::SmartObject& response) const {
-  std::string info;
+  const mobile_apis::FunctionID::eType id =
+      static_cast<mobile_apis::FunctionID::eType>(function_id());
+
+  if (!helpers::Compare<mobile_apis::FunctionID::eType,
+                        helpers::EQ,
+                        helpers::ONE>(
+          id,
+          mobile_apis::FunctionID::SubscribeVehicleDataID,
+          mobile_apis::FunctionID::UnsubscribeVehicleDataID,
+          mobile_apis::FunctionID::GetVehicleDataID,
+          mobile_apis::FunctionID::SendLocationID)) {
+    return;
+  }
 
-  RPCParams::const_iterator it =
-      removed_parameters_permissions_.disallowed_params.begin();
-  for (; it != removed_parameters_permissions_.disallowed_params.end(); ++it) {
-    AddDissalowedParameterToInfoString(info, (*it));
+  std::string disallowed_by_user_info;
+  InfoAdder user_info_adder(disallowed_by_user_info);
+
+  std::for_each(removed_parameters_permissions_.disallowed_params.begin(),
+                removed_parameters_permissions_.disallowed_params.end(),
+                user_info_adder);
+
+  const uint32_t min_number_of_disallowed_params = 1;
```

Comment:
@ZhdanovP Use postfix `u` for unsigned variables

Reply:
@AByzhynar please take a look 49e48c635d6726880b071ee21f85eff7ac8857f0